pkgs/tools/filesystems/dduper/default.nix

```diff
@@ -0,0 +1,51 @@
+{ lib, stdenv, fetchpatch, fetchFromGitHub, btrfs-progs, python3 }:
+
+let
+  btrfsProgsPatched = btrfs-progs.overrideAttrs (oldAttrs: {
+    version = "5.9";
```

Comment:
why do you override the version? Anyway it has no effect with overrideAttrs

Reply:
Does it not? When I build, it shows that it's downloading the `5.9` source bundle (which is what I want).
I override the version because that's what the patch was made for.

Reply:
you are currently patching the nixpkgs version of btrfs-progs, and the patch applies, so all good. It's probably better to keep the newer version of nixpkgs anyway. Overriding version with overrideDerivation or overrideAttrs only changes the name of the derivation, not it's src attribute.